plugins/modules/docker_container.py

```diff
@@ -285,6 +285,8 @@
     description:
       - Dictionary of key,value pairs.
       - Values which might be parsed as numbers, booleans or other types by the YAML parser must be quoted (e.g. C("true")) in order to avoid data loss.
+      - Please note that if you are passing values in with Jinja2 templates, like C("{{ value }}"), you need to add C(| string) to avoid Ansible to
```

Comment:
Feel free to ignore though.
```suggestion
      - Please note that if you are passing values in with Jinja2 templates, like C("{{ value }}"), you need to add C(| string) to prevent Ansible to
```